frontend/src/my_team/MyTeamView.jsx

```diff
@@ -94,21 +111,26 @@ function MyTeamView() {
     await response.json();
   }
 
+  function openScout() {
+    setDisplayScout(true);
+  }
```

Comment:
No need for a separate function definition here if:
1. it's a short function
2. it isn't reused

I'd prefer to inline this instead so like:

`<button onClick={setDisplayScout} />`